clowdapp.yaml

```diff
@@ -0,0 +1,260 @@
+      prometheus.io/port: "9394"
+      prometheus.io/scrape: "true"
+  spec:
+    envName: ${ENV_NAME}
+# We need to add a shared DB definition here ...
+# Perhaps on the defintion of compliance-backend we
+# should create the DB , and make all services depend
+# on it. The "source DB" for every other service must
+# be the one in charge of running DB migrations. I'd not
+# like it to be compliance-prometheus-exporter. That would
+# look VERY weird ...
+#    database:
+#      sharedDbAppName: compliance-backend
+#      version: 12
+#    dependencies:
+#      compliance-backend
+    database:
+      name: compliance
+    inMemoryDb: true
+#    optionalDependencies:
+#    - rbac
+    deployments:
+    - name: service
+      minReplicas: ${{REPLICAS_PROMETHEUS}}
+#      webServices:
+#        public:
+#          enabled: true
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        initContainers:
+        - name: run-migrations
+          image: " "
+          imagePullPolicy: Always
+          command:
+            - bash
+            - -c
+            - |
+              set -e
+              RAILS_ENV="${RAILS_ENV}" bundle exec rake --trace db:debug db:migrate
+          env:
+          - name: RAILS_ENV
+            value: "${RAILS_ENV}"
+          - name: PATH_PREFIX
+            value: "${PATH_PREFIX}"
+          - name: RAILS_LOG_TO_STDOUT
+            value: "${RAILS_LOG_TO_STDOUT}"
+          - name: POSTGRESQL_MAX_CONNECTIONS
+            value: ${POSTGRESQL_MAX_CONNECTIONS}
+#          - name: DATABASE_SERVICE_NAME
+#            value: "${DATABASE_SERVICE_NAME}"
+#          - name: POSTGRESQL_USER
+#            valueFrom:
+#              secretKeyRef:
+#                key: db.user
+#                name: compliance-db
+#          - name: POSTGRESQL_PASSWORD
+#            valueFrom:
+#              secretKeyRef:
+#                key: db.password
+#                name: compliance-db
+#          - name: COMPLIANCE_DB_SERVICE_HOST
+#            valueFrom:
+#              secretKeyRef:
+#                key: db.host
+#                name: compliance-db
+#          - name: POSTGRESQL_DATABASE
+#            valueFrom:
+#              secretKeyRef:
+#                key: db.name
+#                name: compliance-db
+#          - name: SECRET_KEY_BASE
+#            valueFrom:
+#              secretKeyRef:
+#                key: secret_key_base
+#                name: compliance-backend
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /playbooks
+            port: web
+            scheme: HTTP
+          initialDelaySeconds: 35
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 120
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /playbooks
+            port: web
+            scheme: HTTP
+          initialDelaySeconds: 35
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 120
```

Comment:
Can we just use defaults? Like `failureThreshold` is 3 by default already. What's the magic `35` second initialDelay? I think that one is 1 second by default. It should take no time at all to spin up nginx, right?

Reply:
sure, I agree. I commented out most of the stuff - the template however is very on a very WIP stage, we need to update Compliance to be able to read from Clowder config for this template to work at all.